packages/compiler/test/formatter/formatter.test.ts

```diff
@@ -1687,6 +1723,31 @@ alias Foo = [
     });
   });
 
+  describe("empty statements", () => {
+    it("remove empty statements", () => {
+      const foo = 123; /** abc def ghi */ /** abc def ghi */
```

Comment:
```suggestion
```

Should this line just be deleted?  It's causing a lint failure:

```
  1728:13  warning  'foo' is assigned a value but never used. Allowed unused vars must match /^_/u  @typescript-eslint/no-unused-vars
```

https://dev.azure.com/azure-sdk/public/_build/results?buildId=2848158&view=logs&j=04301bb2-aad8-5bfd-762c-8056a87f3309&t=5f153439-c24b-5508-ce0c-d3e9dc372bb4&l=177